docs/wiki/Getting Started With Fuzzing.md

```diff
@@ -0,0 +1,135 @@
+	seeds := [][]byte{
+		[]byte("4kliaOCha2"),
+		[]byte("123456"),
+		[]byte("23Fgfge34"),
+		[]byte("asdfgth#4"),
+	}
+
+	for _, seed := range seeds {
+		f.Add(rand.Intn(100), seed) // Use f.Add to add the seeds above.
+	}
+
+// A fuzz target must be a method call to `(*testing.F).Fuzz` which accepts a
+// `*testing.T` as the first parameter, followed by the fuzzing arguments. There
+// is  no return value. There must be exactly one fuzz target per fuzz test.
+
+// `f.Fuzz` accepts a function and runs fuzzing with randomly generated values 
+// based on the seed corpus provided. All seed corpus entries must have types
+// which are identical to the fuzzing arguments, in the same order. This is true
+// for calls to `(*testing.F).Add`.
+	f.Fuzz(func(t *testing.T, n int, a []byte) {
+		// Inputs can be validated and skipped if they are not suitable.
+		if n > 100000000 {
+			t.Skip()
+		}
+		crypter := NewCrypter(a)
+		thing := randB(n)
+		encThing, err := crypter.Encrypt(thing)
+		if err != nil {
+			t.Fatalf("Encrypt error: %v", err)
+		}
+		reThing, err := crypter.Decrypt(encThing)
+		if err != nil {
+			t.Fatalf("Decrypt error: %v", err)
+		}
+		if !bytes.Equal(thing, reThing) {
+			t.Fatalf("%x != %x", thing, reThing)
+		}
+	})
+}
+```
+
+The fuzzing arguments can only be the following types:
+ - string, []byte
+ - int, int8, int16, int32/rune, int64
+ - uint, uint8/byte, uint16, uint32, uint64
+ - float32, float64
+ - bool
+
+ ## Running a Fuzz Test
+ There are two modes of running fuzz test:
+ - As a unit test (default `go test`). 
+    Fuzz tests are run much like a unit test by default. Each seed corpus entry will be tested against the fuzz target, reporting any failures before exiting. Failed Fuzz inputs generated by the fuzzing engine for that particular fuzz target are re-run against the fuzz target as part of the default `go test`.
+
+ - With fuzzing `go test -fuzz=FuzzTestName`.
+    Each seed corpus entry will be tested against the fuzz target. Also, If there are seed corpus already generated for the fuzz target in `$GOCACHE` they will be run against the fuzz target, and then the fuzzing engine will continue to generate and test seed corpus until `-fuzztime` (See [Fuzz Flags](#Other-Highlighted-Fuzz-Flags)).
+    
+    To enable fuzzing, run go test with the `-fuzz` flag, providing a test name (e.g `FuzzTestName`) or regex matching a single fuzz test. By default, all other tests in that package will run before fuzzing begins. This is to ensure that fuzzing won’t report any issues that would already be caught by an existing test.
+
+    **Note**: Fuzzing cannot be run for multiple packages at the same time using the `-fuzz` flag. You must specify at most one Fuzz test (e.g `-fuzz=FuzzXXX`) (See [suppport for multiple Fuzz tests](https://go.googlesource.com/proposal/+/master/design/draft-fuzzing.md#fuzzing-engine-supports-multiple-fuzz-tests-at-once)).
```

Comment:
From "To enable..." down to here is indented as part of the previous bullet.  Once you get markdownlint running and remove stray spaces it should start to look right.